src/utils/bitcoin-core-client.ts

```diff
@@ -8,7 +8,7 @@ interface RecipientsToUTXOAmounts {
     [key: string]: string;
 }
 export class BitcoinCoreClient {
-    private client: typeof Client;
+    client: typeof Client;
```

Comment:
Just to double-check, is exposing this intentional?

Reply:
Yep, in case I need some other quick functionalities from the bitcoin client in the future I'd like to use them directly rather than open another PR here. If those functionalities become important enough they'll make their way into a PR